spec/requests/graphql/query/batch_imports_spec.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Retrieving Batch Imports', type: :request do
+  context "when logged in user is administrator" do
+    let(:expected_response) do
+      %(
+        [
+          {
+            "fileLocation": "managed-disk://path/to/file",
```

Comment:
How are these tests passing if the expected_response doesn't contain the "id" fields that are requested in the GraphQL query?

Reply:
Is be_json_eql not that exact of a matching method?